src/GitScan/Command/StatusCommand.php

```diff
@@ -89,7 +91,7 @@ protected function execute(InputInterface $input, OutputInterface $output) {
     if (!empty($rows)) {
       $table = $this->getApplication()->getHelperSet()->get('table');
       $table
-        ->setHeaders(array('Status', 'Path', 'Local Branch / Tag', 'Remote Branch'))
+        ->setHeaders(array('Status', 'Path', 'Local Branch / Tag', 'Remote Branch', 'Upstream'))
```

Comment:
Some `git` tutorials teach people to create remotes with the literal names of `origin` and `upstream`. Since this is actually displaying the `origin` URL, the heading should probably be "Origin" or the more generic "Remote URL" ?